CMakeLists.txt

```diff
@@ -29,6 +29,41 @@ endif()
 set(dronecore_install_include_dir "include/dronecore")
 set(dronecore_install_lib_dir ${lib_path})
 
+# We need tinyxml2 for the camera definition parsing.
```

Comment:
Just to comment on comments (meta-commenting :D): this comment will get outdated as soon as we use tinyxml2 for another purpose than the camera definition parsing. So I will probably never trust it.

If at some point I question the need for tinyxml2, I will make my own research: look for includes for tinyxml2 in the code, or try to remove the dependency and see where the build breaks. This comment doesn't add value to me, as in the process I will realize that tinyxml2 is used for the camera, indeed.

Reply:
I suppose so.